core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -863,6 +863,16 @@ static RelDataType deriveTypeSplit(SqlOperatorBinding operatorBinding,
           OperandTypes.STRING.or(OperandTypes.BINARY),
           SqlFunctionCategory.STRING);
 
+  /** The "TO_CHAR(timestamp, format)" function;
+   * converts {@code timestamp} to string according to the given {@code format}.
+   */
+  @LibraryOperator(libraries = {MYSQL, ORACLE, POSTGRESQL})
+  public static final SqlFunction TO_CHAR =
```

Comment:
PostgreSQL has 3 different to_char:
`to_char ( timestamp, text ) → text`
`to_char ( interval, text ) → text`
`to_char ( numeric_type, text ) → text`

You implemented only it for timestamp type. Can you specify it in jira ticket